docs/get_started.md

```diff
@@ -37,6 +37,8 @@ In addition, at least one Cloud Proxy (also version 8.10 or later) must be set u
 * Git 2.35.0 or later. Updating to the latest stable version is recommended.
   For instructions in installing git, go to [Git's installation documentation](https://git-scm.com/downloads),
   and follow the instructions provided for your operating system.
+* Java 17 or later (Only for Java Adapters).
+  For the latest release of Java visit [Java Downloads](https://www.oracle.com/java/technologies/downloads/)
```

Comment:
I think we want to be more specific here:
* Even for Java Adapters, it's not required, just recommended (for ease of development)
* We want the JDK (rather than just the JRE)
* Versions above 17 may have features that won't work in our server. We could say something like "The latest release of Java 17"

Reply:
Also, let's link to the Zulu distribution we use in the container. In addition to being the same version, I believe the license is more permissive than the Oracle version.